### PR TITLE
Description of the script usage

### PR DESCRIPTION
**Pull request by AlexandrStadolnik**
Adds description of how to install Bakery using script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# UI-Bakery on-prem installation
+
+This document describes how to deploy ui-bakery on-prem via `install_all.sh` script.
+
+:warning: The script installs docker and doker-compose, which may upgrade some dependencies under the hood. Please, be advised that if you are running this script on the OS used as a severs for other applications, those applications may break due that potential dependencies upgrade.
```
Comment (Tibing) [inline README.md:5]: The script installs docker and docker-compose, which may upgrade some dependencies under the hood. Please be advised that if you run this script on the OS used as a server for other applications, those applications may break due to that potential dependencies upgrade.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Must have full rights to use "sudo".
+
+## Installation steps
+
+1. Run this command preferably from `/home` Linux directory to download, install and launch "ui-bakery":
+   ```
+   curl -k -L -o install_all.sh https://raw.githubusercontent.com/uibakery/onpremise/main/install_all.sh && bash ./install_all.sh
+   ```
+1. In the process, upon request, enter the previously received license code, hosting URL and port.
+1. Once the installation is completed, open the browser using URL and provided earlier. By default it is [http://localhost:3030/](http://localhost:3030/).
```
Comment (Tibing) [inline README.md:19]:
```suggestion
2. Once the installation is completed, open the browser using the URL provided earlier. By default it is [http://localhost:3030/](http://localhost:3030/).
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- OS Linux (Ubuntu) 18.04 and above.
+- Must have full rights to use "sudo".
+
+## Installation steps
+
+1. Run this command preferably from `/home` Linux directory to download, install and launch "ui-bakery":
+   ```
+   curl -k -L -o install_all.sh https://raw.githubusercontent.com/uibakery/onpremise/main/install_all.sh && bash ./install_all.sh
+   ```
+1. In the process, upon request, enter the previously received license code, hosting URL and port.
```
Comment (Tibing) [inline README.md:18]:
```suggestion
1. In the process, upon request, enter the previously received license code, hosting URL and port.
```

**Review by Tibing — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AlexandrStadolnik) [reply README.md:18]: Says this suggestion is invalid because no changes were made to the code

File: README.md (hunk 2 above)
Comment (AlexandrStadolnik) [reply README.md]: Resolved

**Review by Tibing — APPROVED** [review]

**Pull request merged**